### ISSUE TITLE
"wc-checkout-draft" status shown as WC default status in Custom Order Status Tool section.

### ISSUE THREAD

**Issue by apeksha10** [TycheSoftwares/custom-order-statuses-woocommerce-lite#28]
"wc-checkout-draft" status shown as WC default status in Custom Order Status Tool section.

WooCommerce does not provide such kind of status. I took latest git copy and after uploading that copy on my localhost, Custom Order Status Tool section shows this status as default.

see the screenshot : https://www.screencast.com/t/9mqhmyW7kjc 

We should delete this status.

**Comment by ashokrane** [comment]
@apeksha10 Which WooCommerce version you are using at your end? This status is not coming from any other plugin?

**Comment by apeksha10** [comment]
I am using WC 4.3.1 version. https://www.screencast.com/t/yH12aw8vc
Activate plugin list : https://www.screencast.com/t/bdxYvKV3hOb

**Comment by ashokrane** [comment]
Deactivate the Blocks plugin & check.

**Comment by nitin-ts** [comment]
Hello,

I have checked it in fresh WordPress install and it is showing that status if I installed an older version of WooCommerce and if I update it to the latest then it doesn't show me that again.

With below active plugins, it shows me wc-checkout-draft status.

![image](https://user-images.githubusercontent.com/67450817/89002955-126e8780-d31c-11ea-8caf-ea5232fc2cd5.png)

![image](https://user-images.githubusercontent.com/67450817/89002982-24e8c100-d31c-11ea-9733-09c59165fecd.png)

And when I update it to the latest WooCommerce, then it doesn't show me that status.

![image](https://user-images.githubusercontent.com/67450817/89003033-55305f80-d31c-11ea-8a04-278dfed30f10.png)

![image](https://user-images.githubusercontent.com/67450817/89003055-60838b00-d31c-11ea-8570-3a8bfef27ecb.png)

In short, it is coming from the WooCommerce plugin.

I have seen that @apeksha10 has that status even if she has the latest WooCommerce plugin but I don't have it when I have the latest WooCommerce plugin.

Please let me on this.

**Comment by apeksha10** [comment]
Yes.. with Latest WC version this issue is not replicate.

Hence closed this issue and marked as fixed.
